controllers/seasons.controllers.ts

```diff
@@ -1,7 +1,116 @@
-// TODO: require the model
-// Season = require("../models/Season")
+import { NextFunction, Response, Request } from "express";
+import Season from "../models/Season";
 
-// TODO: add controller logic 
+function create(req: Request, res: Response, next: NextFunction) {
+  const season = new Season({
+    name: req.body.name,
+    startDate: req.body.startDate,
+    endDate: req.body.endDate, //Think this should be calculated when status is changed to close
+    payroll_timeframe: req.body.payroll_timeframe,
```

Comment:
Let's not mix up camel case and snake case, I also was confused at first, but let's change this to camel case.
Also, let's set this defaulted to Byweekly

Reply:
https://github.com/LangaraVoyagers/project-2-api/pull/8/commits/bda999083f5d89af1e9bfa8b076b19880029b043

Reply:
https://github.com/LangaraVoyagers/project-2-api/pull/8/commits/c70eeb511a6abba27246f741b9131a010014f958